cypress/integration/tests/applicationinventory/application/sort.test.ts

```diff
@@ -0,0 +1,125 @@
+        // Create multiple applications with tags
+        for (let i = 0; i < 3; i++) {
+            // Create new application
+            const application = new ApplicationInventory(data.getFullName(), "", "", "", tagsList);
+            application.create();
+            applicationsList.push(application);
+            tagsList.pop();
+        }
+    });
+
+    beforeEach("Persist session", function () {
+        // Save the session and token cookie for maintaining one login session
+        Cypress.Cookies.preserveOnce("AUTH_SESSION_ID", "KEYCLOAK_SESSION");
+
+        // Interceptors
+        cy.intercept("GET", "/api/application-inventory/application*").as("getApplications");
+    });
+
+    after("Perform test data clean up", function () {
+        // Delete the applications created before the tests
+
+        applicationsList.forEach(function (application) {
+            application.delete();
+        });
+    });
+
+    it("Name sort validations", function () {
+        // Navigate to application inventory page
+        clickByText(navMenu, applicationinventory);
+        cy.wait("@getApplications");
+
+        // get unsorted list when page loads
+        const unsortedList = getTableColumnData(name);
+
+        // Sort the application inventory by name in ascending order
+        sortAsc(name);
+        cy.wait(2000);
+
+        // Verify that the application inventory table rows are displayed in ascending order
+        const afterAscSortList = getTableColumnData(name);
+        verifySortAsc(afterAscSortList, unsortedList);
+
+        // Sort the application inventory by name in descending order
+        sortDesc(name);
+        cy.wait(2000);
+
+        // Verify that the application inventory table rows are displayed in descending order
+        const afterDescSortList = getTableColumnData(name);
+        verifySortDesc(afterDescSortList, unsortedList);
+    });
+
+    it("Review sort validations", function () {
+        // Navigate to application inventory page
+        clickByText(navMenu, applicationinventory);
+        cy.wait("@getApplications");
+
+        // get unsorted list when page loads
+        const unsortedList = getTableColumnData(review);
+
+        // Sort the application inventory by review in ascending order
+        sortAsc(review);
+        cy.wait(2000);
+
+        // Verify that the application inventory table rows are displayed in ascending order
+        const afterAscSortList = getTableColumnData(review);
+        verifySortAsc(afterAscSortList, unsortedList);
+
+        // Sort the application inventory by name in descending order
+        sortDesc(review);
+        cy.wait(2000);
+
+        // Verify that the application inventory table rows are displayed in descending order
+        const afterDescSortList = getTableColumnData(review);
+        verifySortDesc(afterDescSortList, unsortedList);
+    });
+
+    it("Tag count sort validations", function () {
```

Comment:
For this a small wait might be required after page items are selected, so I would suggest placing a wait at start after application inventory page loads. The logic is fine